common/app/views/fragments/commercial/contentLogo.scala.html

```diff
@@ -1,11 +1,10 @@
 @(branding: common.commercial.Branding)
 
-<div class="brandbadge brandbadge--onside ad-slot--force-display ad-slot--paid-for-badge--interactive">
-    <div class="brandbadge__inner">
+<div class="brandbadge brandbadge--onside ad-slot--force-display ad-slot--paid-for-badge ad-slot--paid-for-badge--front">
```

Comment:
Good points. I changed the markup temporary just to check how works and it broke badges everywhere ;/. As I don't have time for refactoring styles and test it everywhere on the page, I will make those changes later in another PR.